sc-locked-asset-factory/src/lib.rs

```diff
@@ -128,11 +136,13 @@ pub trait LockedAssetFactory:
             let new_unlock_schedule = UnlockSchedule {
                 unlock_milestones: new_unlock_milestones,
             };
+            let do_cache_result = false;
```

Comment:
does this mean if if I unlock the same type of locked mex two times I get two different nonces ? like I have 1000 locked mex, and I do 2 transactions of unLock of 500, each time I need to get 10% real mex and the rest to be locked mex, I get 2 types of locked mex ?

Reply:
yes. merging and caching are not the best combination. decided that it is best not to cache tokens after they have been unlocked. it's crucial to have them cached at least before unlocking, like when receiving rewards from farm or distribution. those are tokens with a standard unlock schedule. once they are unlocked, because both unlocking time and amount combine to generate a new unlock schedule, it can result in an infinite combination and an infinite different schedule.